modules/ROOT/pages/change-runtime-version.adoc

```diff
@@ -0,0 +1,19 @@
+= Change the Version of Mule Runtime that a Mule App Uses
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can change an existing Mule application to use a newer version of Mule Runtime. You might want to do this if the newer version includes updates or new features that you want your application to take advantage of.
```

Comment:
Apparently, it is now Mule runtime engine (lowercase runtime engine). Or just Mule. 

But not Mule Runtime

Reply:
If we are using SF style guide, for versioning, they use "earlier" and "later."